packages/yoga/src/Card/native/PlanCard/PlanCard.test.jsx

```diff
@@ -1,141 +1,73 @@
-    });
+  const renderPlan = () =>
+    render(
+      <ThemeProvider>
+        <PlanCard>
+          <PlanCard.Tag>Recommended Plan</PlanCard.Tag>
+          <PlanCard.Content
+            subtitle="plan"
+            title="Basic"
+            currency="$"
+            price="99.90"
+            period="/month"
+          >
+            <PlanCard.Subtitle>Get access to</PlanCard.Subtitle>
+            <PlanCard.List>
+              <PlanCard.ListItem
+                icon={Star}
+                text="list item"
+                buttonProps={{
+                  children: 'button',
+                  onPress: buttonOnPressMock,
+                }}
+              />
+            </PlanCard.List>
+          </PlanCard.Content>
+          <PlanCard.Actions>
+            <Button full>Select this plan</Button>
+          </PlanCard.Actions>
+        </PlanCard>
+      </ThemeProvider>,
+    );
 
-    it('should match snapshot with tertiary PlanCard', () => {
-      const { container } = render(
-        <ThemeProvider>
-          <PlanCard variant="tertiary">
-            <PlanCard.Header>Gympass PlanCard</PlanCard.Header>
-            <PlanCard.Content
-              title="Gympass Black for employees with Buddha Spa + Reebook included"
-              price="U$ 99.90"
-              period="/month"
-            >
-              content&apos;s children is rendered
-            </PlanCard.Content>
-            <PlanCard.Actions>
-              <Button>should have buttons</Button>
-            </PlanCard.Actions>
-          </PlanCard>
-        </ThemeProvider>,
-      );
-      expect(container).toMatchSnapshot();
-    });
+  describe('Events', () => {
+    it('should call onPress when ListItem has a button', () => {
+      const { getByText } = renderPlan();
+      fireEvent.press(getByText('button'));
 
-    it('should match snapshot PlanCard with ribbon', () => {
-      const { container } = render(
-        <ThemeProvider>
-          <PlanCard
-            ribbon={{
-              text: 'Label',
-            }}
-          />
-        </ThemeProvider>,
-      );
-      expect(container).toMatchSnapshot();
-    });
-
-    it('should match snapshot PlanCard with secondary ribbon', () => {
-      const { container } = render(
-        <ThemeProvider>
-          <PlanCard
-            ribbon={{
-              text: 'Label',
-              variant: 'secondary',
-            }}
-          />
-        </ThemeProvider>,
-      );
-      expect(container).toMatchSnapshot();
+      expect(buttonOnPressMock).toHaveBeenCalled();
     });
+  });
 
-    it('should match snapshot PlanCard with tertiary ribbon', () => {
-      const { container } = render(
-        <ThemeProvider>
-          <PlanCard
-            ribbon={{
-              text: 'Label',
-              variant: 'tertiary',
-            }}
-          />
-        </ThemeProvider>,
-      );
-      expect(container).toMatchSnapshot();
+  describe('Snapshots', () => {
```

Comment:
Should we have one snapshot for each variant?

Reply:
I don't think so, if I create a test for each variant it will be testing the same logic.